docs/source/teams/cloud_media.rst

```diff
@@ -652,8 +652,40 @@ collection being annotated, then you can simply pass `cloud_manifest=True`:
     The cloud manifest file must contain all media files in the sample
     collection being annotated.
 
+.. _teams-annotating-cloud-media-v7:
+
+Annotating cloud-backed datasets with V7 Darwin
+__________________________________________
```

Comment:
nit: can you please increase the length of the underscore to match the text above? As is, this will add another red error to the build 😄